### ISSUE TITLE
ScreenReader does not announce current line position when pressing Ctrl+G in VS Code: A11y_Visual Studio Code Client_ScreenReader.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#280889]
### GitHub Tags:
#A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #Win32; #Benchmark;#A11yMAS;#WCAG1.3.1;#Info and Relationships;#ScreenReader;#JAWS;#A11yeDAD

### Environment and OS details:
Application Name: Visual Studio Code
Version: 1.107.0-insider (user setup)

### Reproduction Steps:

1. Open Visual studio code insider.
2. Tab till code editor
3. Press Ctrl+ G to go to the desired line.
4. Observed that ScreenReader does not announce current line position when pressing Ctrl+G in VS Code.

### Actual Result:
ScreenReader does not announce current line position when pressing Ctrl+G in VS Code.

### Expected Result:
When pressing Ctrl+G to navigate to a specific line in Visual Studio Code, the screen reader should announce the current line position. For example:
"Current line: 25."

### User Impact:
Users relying on screen readers cannot determine their current line position when invoking the Ctrl+G command. This creates confusion and makes it difficult to navigate accurately within the code..

### Attachment:
Issue is not repro'ing with the NVDA.
https://github.com/user-attachments/assets/59319eb0-86e7-4ee9-a00a-c0ff61e2b212

Issue is repro'ing with the JAWS
https://github.com/user-attachments/assets/f00768f5-baf9-47a8-b1d6-c7c3beb18d28
